### DISCUSSION TITLE
benthos bloblang required: expected comma or closing bracket

### DISCUSSION THREAD

**Discussion by helloworld6688** [redpanda-data/connect#2101] [General]
i met this problem in my code, but i don't know how to fix it ,so i hope someone who can help me, best wish for you

level=error msg="Config lint error" @service=benthos lint="/benthos.yaml(158,15) line 1 char 1: failed to parse import './mapping/business.blobl'\n |\n1 | import "./mapping/business.blobl"\n | ^---\n\n./mapping/business.blobl: line 18 char 22: required: expected comma or closing bracket\n |\n22 | root.business = "hardware" |

this is benthos config, and my yaml config is ok

```
      - label: hardware
        mapping: |
          import "./mapping/business.blobl"

          root = this
          root.full_data = {}
          root.full_data.info = this.full_data.apply("business")

  business.blobl: |-
    import "common.blobl"
    map business {
      root.source = "hard"
      root.business = "hardware"
    }
```

**Comment by Jeffail** [comment]
Hey @helloworld6688, can you provide a short config example that I can run locally? Otherwise it's hard to tell with all the omitted data. For example, what's contained within `common.blobl`? Converting to discussion as per https://github.com/benthosdev/benthos/issues/2026
